animal-sniffer-signature/build.gradle.kts

```diff
@@ -27,7 +27,7 @@ configurations.add(signatureJarClasspath)
 configurations.add(generatedSignature)
 
 dependencies {
-  signature("com.toasttab.android:gummy-bears-api-21:0.6.1@signature")
+  signature("com.toasttab.android:gummy-bears-api-21:0.7.0@signature")
```

Comment:
Looks like the `gummy-bears` repo may have added support for `desugar_jdk_libs` 2.x: https://github.com/open-toast/gummy-bears/commit/08bae3144792dfe4a95d8f17f664fa04148d965a#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R78

I still think its good for us to maintain our own version of the signature.

Reply:
cc @LikeTheSalad

Reply:
Yeah, they added `2.0.4` corelib support in their latest release. The functionality should remain the same whether we keep our own version or use the coreLib version from `gummy-bears`, I guess really the main benefit of using our own would be that we can keep up-to-date quicker with newer android corelib releases. Apart from that, I don't have strong preferences over any approach.

Btw the reason why I kept the `non-coreLib` version of `gummy-bears` as a dependency is to get Android OS's runtime classes for API 21 in the signature, which should essentially be JDK7 + some android-specific classes, [as listed here](https://developer.android.com/reference/packages).